net/iperf3/files/etc/config/iperf3

```diff
@@ -0,0 +1,16 @@
+
+config server wan
+	option enabled '1'
+	option port '5202'
+	option iface 'wan'
+	option redirect '1'
+	option interval '1'
+
```

Comment:
Maybe in a future version you could add an option for logging, given that it's already possible to capture the output thru redirect.

Reply:
Ok I will do that in the next iteration